contracts/governance/Vesting/VestingStorage.sol

```diff
@@ -40,4 +40,7 @@ contract VestingStorage is Ownable {
 
     /// @notice Constant used for computing the vesting dates.
     uint256 constant FOUR_WEEKS = 4 weeks;
+
+    /// @dev max iterations that can be supported in 1 tx for the withdrawal
+    uint256 internal maxVestingWithdrawIterations;
```

Comment:
this can be queried from the Staking contract instead of keeping it in each Vesting contract

Reply:
Solved in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/f938b80ff498b0936d322885fd6e5d247407339b)